content/2_Bronze/Intro_Greedy.mdx

```diff
@@ -107,18 +107,18 @@ int main() {
 <JavaSection>
 
 From the
-[official analysis](http://www.usaco.org/current/data/sol_breedflip_bronze_feb20.html):
+[official analysis](http://www.usaco.org/current/data/sol_breedflip_bronze_feb20.html)
+(with Kattio added):
 
 ```java
 import java.io.*;
 import java.util.*;
 public class Solution {
 	public static void main(String[] args) throws IOException {
-		BufferedReader br = new BufferedReader(new FileReader("breedflip.in"));
-		PrintWriter pw = new PrintWriter(new BufferedWriter(new FileWriter("breedflip.out")));
-		int n = Integer.parseInt(br.readLine());
-		char[] a = br.readLine().toCharArray();
-		char[] b = br.readLine().toCharArray();
+		Kattio io = new Kattio("breedflip");
+		int n = io.nextInt();
+		char[] a = io.next().toCharArray();
+		char[] b = io.next().toCharArray();
 		int ret = 0;
 		while(!new String(a).equals(new String(b))) {
```

Comment:
Would adding spacing to the while and for loops make it look better?
```suggestion
		while (!new String(a).equals(new String(b))) {
```

Reply:
And also on lines 126, 128, 129, and 130 (between the if statement and the parenthesis)

Reply:
It seems that having no spacing there is a common styling pattern for c++ and sometimes java. I agree that having a space does make it look better, @maggie-j-liu should I change it or leave it?

Reply:
Up to you, I prefer having a space but as long as the code is consistent, it's fine.

Reply:
Alright I can add spaces to make the code consistent with everything else then